org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_RunnableDevice.java

```diff
@@ -32,15 +32,14 @@
 import org.slf4j.LoggerFactory;
 
 class _RunnableDevice<M> extends _AbstractRemoteDevice<M> implements IRunnableDevice<M>, IActivatable {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)